test/lib/action_cable/subscription_adapter/solid_cable_test.rb

```diff
@@ -159,4 +167,11 @@ def subscribe_as_queue(channel, adapter = @rx_adapter)
     ensure
       adapter.unsubscribe(channel, callback) if subscribed.set?
     end
+
+    def with_active_record_logger(logger)
+      old_logger, @server.config.logger = @server.config.logger, logger
```

Comment:
nit but lets expand this out to make it a bit easier to read:
```suggestion
      old_logger = @server.config.logger
      @server.config.logger = logger
```